StackExchange.Redis/StackExchange/Redis/GeoEntry.cs

```diff
@@ -0,0 +1,144 @@
+
```

Comment:
Fundamental question: what, outside of `GeoEntry.cs` uses `GeoEntry`? None of the APIs do...